src/main/clojure/opa4j/builtins/array.clj

```diff
@@ -25,8 +25,8 @@
       (throw (Exception. (format "arg 1 is not an array"))))
     (clojure.core/concat a b)))
 
-(defn reverse [args]
+(defn builtin-reverse [_]
```

Comment:
The `builtin-` prefix is there to not clash with Clojure core functions?

Reply:
Yes. At first I thought it wouldn't matter, but the linter complained. Perhaps this is a category of warning that is ok to suppress, as that prefix bugs me some ..